client/src/pages/Details.js

```diff
@@ -78,25 +154,25 @@ export default function Details({ member, updateMember, members }) {
         {isUnfolded && (
           <>
             <label htmlFor="date">Date</label>
```

Comment:
The following input needs to have an id for `htmlFor` to work

Reply:
Thank you!!